libs/labelbox/tests/data/annotation_import/conftest.py

```diff
@@ -194,6 +193,21 @@ def llm_human_preference_data_row(global_key):
     return llm_human_preference_data_row
 
 
+@pytest.fixture(scope="module")
+def mmc_data_row_url():
+    return "https://storage.googleapis.com/labelbox-datasets/conversational_model_evaluation_sample/offline-model-chat-evaluation.json"
+
+
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
why `autouse=True` ? doesn't it mean the fixture will be request by each and every test in annotation_import?

Reply:
I was inspired by what @Gabefire did on other similar fixtures. Definitely doesn't have to be autouse, but I guess it makes the fixtures run once at session start and cache results?